### PR TITLE
update README

### PR DESCRIPTION
**Pull request by Surendrajat**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [ ] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [ ] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**
I've tried to tidy up README a bit.

**Test plan (required)**
See if it improves README.

**Closing issues**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,47 +2,49 @@
 
 # Cutter
 
-Cutter is a free and open-source reverse engineering framework powered by [rizin](https://github.com/rizinorg/rizin) . Its goal is making an advanced, customizable and FOSS reverse-engineering platform while keeping the user experience in mind. Cutter is created by reverse engineers for reverse engineers.  
+Cutter is a free and open-source reverse engineering platform powered by [rizin](https://github.com/rizinorg/rizin). It aims at being an advanced and customizable RE platform while keeping the user experience in mind. Cutter is created by reverse engineers for reverse engineers.
```
Comment (karliss) [inline README.md:5]: Abbreviating RE only makes sense if you have a well established context context, but establishing context is the main purpose of first paragraph. Initial description describing what this should be as clear as possible.

File: README.md [hunk 2]
```diff
@@ -2,47 +2,49 @@
...
+### Download
 
-Cutter is available for all platforms (Linux, macOS, Windows).
-You can download the latest release [here](https://github.com/rizinorg/cutter/releases).
- * macOS: Download the latest `.dmg` file or use [Homebrew Cask](https://github.com/Homebrew/homebrew-cask) `brew cask install cutter`.
- * Windows: Download the latest Zip archive.
- * Linux: Download the latest AppImage file. Then just make it executable and run it:
-   * `chmod +x <appimage_file>`
-   * `./<appimage_file>`
+Cutter release binaries for all major platforms (Linux, macOS, Windows) can be downloaded from [GitHub Releases](https://github.com/rizinorg/cutter/releases).
```
Comment (karliss) [inline README.md:18]: This was useful information. I don't it should be removed.

File: README.md (hunk 1 above)
Comment (Surendrajat) [reply README.md:5]: I abbreviated it for similar reason. The first sentence has the 'reverse engineering platform' for cutter. Felt like repeating.

File: README.md (hunk 2 above)
Comment (Surendrajat) [reply README.md:18]: okay. Is the homebrew thingy officially supported?

**Comment by xarkes** [comment]
I agree with Karliss remarks, other than that, looks good to me.

File: README.md [hunk 3]
```diff
@@ -2,47 +2,49 @@
...
+
+### [Contribution Guidelines](https://cutter.re/docs/contributing.html)
+
+### [Developers Docs](https://cutter.re/docs/contributing/code.html)
 
 ## Plugins
-Cutter supports both Python and Native C++ plugins. Want to extend Cutter with Plugins? Read the [Plugins](https://cutter.re/docs/plugins) section on our documentation.
+Cutter supports both Python and Native C++ plugins.
+
+Our community has built many plugins and useful scripts for Cutter such as the native integration of [Ghidra](https://ghidra-sre.org) decompiler or the plugin to visualize DynamoRIO code coverage. You can find a list of cutter plugins linked below. Feel free to extend it with your own plugins and scripts for Cutter.
```
Comment (ITAYC0HEN) [inline README.md:39]: I'd actually link to rz-ghidra and not to Ghidra itself :)

**Comment by ITAYC0HEN** [comment]
that's a good one, thanks!

File: README.md (hunk 3 above)
Comment (Surendrajat) [reply README.md:39]: okay. I thought so too but for that, I guess the whole `integration of Ghidra decompiler` should be a link and I chickened out :P I Will add it :+1:

**Review by ITAYC0HEN — APPROVED** [review]
Thank you!

**Pull request merged**